### DISCUSSION TITLE
Questions about task 2 deadline

### DISCUSSION THREAD

**Discussion by cuteyyt** [FETS-AI/Challenge#41] [Q&A] [answered]
Dear Organizers:
Thank you very much for your hard work in organizing this challenge.

As the homepage only says July 19th is the short paper's deadline, then the challenge will enter the testing phase. I want to figure out whether the model for task 2 is allowed to be submitted after July 19th, or we should submit our model before July 19th. Moreover, can we submit a version on or before July 19th then submit a model again after that day (whether we can submit more than once)?

**Comment by mzenk** [comment, marked as answer]
Hi @cuteyyt,
the deadline on July 19th is for *both* short paper submission and algorithm submission.
I recently updated the [task-2 repo](https://github.com/FETS-AI/Challenge/tree/main/Task_2) with details on the submission. Multiple submission before the deadline are allowed. The last submission will be taken as your final submission for the testing phase.

If you have more questions about the submission, feel free to ask them here :)

**Reply by cuteyyt** [reply]
Thank you for your answer!
